render/egl.c

```diff
@@ -358,6 +358,58 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 	return NULL;
 }
 
+struct wlr_egl *wlr_egl_create_from_drm_fd(int drm_fd) {
+	PFNEGLQUERYDEVICESTRINGEXTPROC eglQueryDeviceStringEXT;
+	PFNEGLQUERYDEVICESEXTPROC eglQueryDevicesEXT;
+
+	load_egl_proc(&eglQueryDevicesEXT, "eglQueryDevicesEXT");
+	load_egl_proc(&eglQueryDeviceStringEXT,"eglQueryDeviceStringEXT");
+
+	static const EGLint MAX_DEVICES = 64;
+	EGLint nb_devices = 0;
+
+	eglQueryDevicesEXT(MAX_DEVICES, NULL, &nb_devices);
```

Comment:
We need to check the return value of `eglQueryDevicesEXT` (there's are more unchecked calls below).

Reply:
Fixed in v2